plugins/cdvdGigaherz/src/ReadThread.cpp

```diff
@@ -136,10 +143,7 @@ DWORD CALLBACK cdvdThread(PVOID param)
             continue;
         }
 
-        if (prefetch_left)
-            WaitForSingleObject(hNotify, 1);
-        else
-            WaitForSingleObject(hNotify, 250);
+        s_notify_cv.wait_for(guard, std::chrono::milliseconds(prefetch_left ? 1 : 250));
```

Comment:
It should be safe? Well, sort of, I think there's actually a pre-existing data race that I haven't actually dealt with (On master, 1.4.0, and this PR I managed to get some TLB misses in Growlanser: Heritage Of War when using the plugin, but it works perfectly fine on ISO).